src/components/Repository.js

```diff
@@ -125,10 +163,15 @@ function Repository({match}) {
               <a rel="noreferrer" target="_blank" href={`https://codetriage.com/${nameWithOwner}`}>
                 <Button primary>Set up CodeTriage</Button>
               </a>
+              {isForked ?
+                <a rel="noreferrer" target="_blank" href={`https://github.com/${user.login}/${repoName}`}>
+                  <Button disabled={isForkLoading} data-test="go-to-fork-button">Go to fork</Button>
```

Comment:
https://github.com/open-sauced/open-sauced/issues/743#issuecomment-650698602

```suggestion
                  <Button disabled={isForkLoading} data-test="go-to-fork-button">View fork</Button>
```